### PR TITLE
Add `augmentation.ts` to the TypeScript package generator.

### PR DESCRIPTION
**Pull request by martnpaneq**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Feature: Add `augmentation.ts` to the TypeScript package generator. Closes ckeditor/ckeditor5-package-generator#144.

---

### Additional information

*For example – encountered issues, assumptions you had to make, other affected tickets, etc.*

### Supported scopes

* `generator` → https://www.npmjs.com/package/ckeditor5-package-generator
* `*` → https://www.npmjs.com/package/@ckeditor/ckeditor5-package-*

### CODE REVIEW

File: packages/ckeditor5-package-generator/lib/templates/ts/src/_PLACEHOLDER_.ts.txt [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import { Plugin } from 'ckeditor5/src/core';
 import { ButtonView } from 'ckeditor5/src/ui';
+import './augmentation';
```
Comment (psmyrek) [inline packages/ckeditor5-package-generator/lib/templates/ts/src/_PLACEHOLDER_.ts.txt:3]: Shouldn't this be imported by `index.ts` instead of plugin?

**Review by psmyrek — APPROVED** [review]
LGTM 🎉

File: packages/ckeditor5-package-generator/lib/templates/ts/sample/index.html [hunk 2]
```diff
@@ -72,6 +72,7 @@ <h3>The directory structure</h3>
 │  └─ ckeditor.ts      # The editor initialization script.
 ├─ src
 │  ├─ <%= formattedNames.plugin.lowerCaseMerged %>.ts
+│  ├─ augmentation.ts  # Type augmentations for the `@ckeditor/ckeditor5-core` module. Read more in <a href="https://ckeditor.com/docs/ckeditor5/latest/api/module_core_plugincollection-PluginsMap.html">PlugginsMap</a> and <a href="https://ckeditor.com/docs/ckeditor5/latest/api/module_core_commandcollection-CommandsMap.html">CommandsMap</a>.
```
Comment (pomek) [inline packages/ckeditor5-package-generator/lib/templates/ts/sample/index.html:75]:
```suggestion
│  ├─ augmentation.ts  # Type augmentations for the `@ckeditor/ckeditor5-core` module. Read more in <a href="https://ckeditor.com/docs/ckeditor5/latest/api/module_core_plugincollection-PluginsMap.html">PluginsMap</a> and <a href="https://ckeditor.com/docs/ckeditor5/latest/api/module_core_commandcollection-CommandsMap.html">CommandsMap</a>.
```

**Review by pomek — APPROVED** [review]
LGTM.

**Pull request merged**